crypto/key_not_openssl.go

```diff
@@ -0,0 +1,40 @@
+// +build !openssl
+
+package crypto
+
+import (
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/rsa"
+
+	btcec "github.com/btcsuite/btcd/btcec"
+	"golang.org/x/crypto/ed25519"
+)
+
+// KeyPairFromKey generates a new private and public key from an input private key
```

Comment:
Godoc is a bit bleak. “Generates” is an imprecise term here, as there’s no key generation happening, rather wrapping or conversion.

Reply:
done